### PR TITLE
Get openshift-vpa ns access to group neu-students

### PR DESCRIPTION
**Pull request by apoorvam**
Changes:
* Gives `openshift-vertical-pod-autoscaler` namespace access to `neu-students` group. This helps in accessing logs of the operator for debugging purposes.

### CODE REVIEW

**Review by 4n4nd — COMMENTED** [review]
/lgtm

**Comment by apoorvam** [comment]
@HumairAK Just updated this to use view role binding. Please review.

File: cluster-scope/components/project-view-rolebindings/neu-students/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+    name: namespace-view-neu-students
+roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: view
+subjects:
+    - apiGroup: rbac.authorization.k8s.io
```
Comment (HumairAK) [inline cluster-scope/components/project-view-rolebindings/neu-students/rbac.yaml:10]: can you match the indents with the rest of the file?

**Pull request merged**